docker/DockerFile

```diff
@@ -8,6 +8,8 @@ RUN yum -y install yum-utils sudo which vim net-tools strace lsof tcpdump openss
 RUN mkdir -m 600 /root/.ssh
 #TODO: shouldn't hardcode the private key
 RUN echo -e '_REPLACE_WITH_YOUR_PRIVATE_KEY_' > /root/.ssh/id_rsa
+RUN echo -e '_REPLACE_WITH_YOUR_PUBLIC_KEY_' > /root/.ssh/authorized_keys
```

Comment:
Can we leave DockerFile as it is please?
Readonly / deprecated (no more change)